src/NHibernate/nhibernate-configuration.xsd

```diff
@@ -15,7 +15,6 @@
 						<xs:documentation>
 							There are 3 default short-cut values
 							- lcg     : default for .NET2.0 and higher.
-							- codedom : CodeDOM-based bytecode provider (mostly for .NET1.1).
 							- null    : Disable the reflection optimization completely.
```

Comment:
`There are 3` should be changed to `There are 2`.

Reply:
Fixed.